src/app/hooks/useTotalBalance.ts

```diff
@@ -0,0 +1,16 @@
+import { useAppSelector } from './reduxHooks';
+import { useState } from 'react';
+import { Balance, Currency } from 'app/interfaces/common/Balance';
+
+export const useTotalBalance = (currency: Currency) => {
+  const {
+    deposit: { balances },
+  } = useAppSelector(state => state);
+  const [totalBalance] = useState<number>(
```

Comment:
Why do we need a useState here?

Reply:
Removed hook, replaced with const on the components using this hook.